.github/workflows/galaxy-quick-impl.yaml

```
@@ -16,22 +16,14 @@ on:
         required: false
         type: string
         default: "in-service"
-      wormhole_b0:
-        required: false
-        type: boolean
-        default: true
       blackhole:
         required: false
         type: boolean
-        default: true
-      topology-6u:
-        required: false
-        type: boolean
-        default: true
+        default: false
 
 jobs:
   quick-wh-glx-health:
-    if: ${{ inputs.topology-6u && inputs.wormhole_b0 }}
+    if: ${{ (inputs.bh == 'false') && (github.event_name != 'schedule') }}
```

Would be more readable if the condition was flipped IMO

```suggestion
    if: ${{ github.event_name == 'schedule' || !inputs.blackhole }}
```